uber/templates/registration/new_base.html

```diff
@@ -122,7 +195,7 @@
 <br/>
 
 {% block table %}
-<table class="table datatable" data-page-length="-1">
+<table class="table datatable table-striped" data-page-length="10">
```

Comment:
I noticed that loading up several stripe forms takes more time than putting them on another page, so I limited the number of registrations shown. I don't actually know how to run the profiler, but I'd love to see if the performance is a really big hit. If so, we may have to figure out a way to not load the stripe form on this page.